src/Commands/Backup/Automatic/InfoCommand.php

```diff
@@ -28,12 +29,12 @@ class InfoCommand extends TerminusCommand implements SiteAwareInterface
      * @default-string-field weekly_backup_day
      * @return PropertyList
      *
-     * @param string $site_env Site & environment to get the schedule of, in the format `site-name.env`.
+     * @param string $site_env Site & environment in the format `site-name.env`
      *
      * @usage terminus backup:automatic:info <site>.<env>
-     *     Responds with the day of the week backups are scheduled for on the <env> environment of <site>
-     * * @usage terminus backup:automatic:info <site>.<env> --format=table
-     *     Responds with the day of the week and hour of the day backups are scheduled for on <site>.<env>
+     *     Displays day of the week whose backup is retained for 1 month.
```

Comment:
The text on this command's usage doesn't include <site> and <env> like the others do.